docs/docs/building-applications/1-grabbing-images.md

```diff
@@ -1,3 +1,7 @@
+---
```

Comment:
I explicitly gave each of the docs in /building-applications a place in the sidebar list by prepending each one with `<{position}->` and adding `sidebar_position: {position}` where necessary.